app/controllers/microposts_controller.rb

```diff
@@ -0,0 +1,34 @@
+class MicropostsController < ApplicationController
+  before_action :logged_in_user, only: %i(create destroy)
+  before_action :correct_user, only: :destroy
+
+  def create
+    @micropost = current_user.microposts.build micropost_params
+    @micropost.image.attach micropost_params[:image]
+    if @micropost.save
+      flash[:success] = t ".success_mess"
+      redirect_to root_path
+    else
+      flash.now[:danger] = t ".fail_mess"
+      @feed_items = current_user.feed.paginate page: params[:page]
```

Comment:
change 
` @feed_items = current_user.feed.paginate page: params[:page]`
to 
` @feed_items = current_user.feed.recent_posts.paginate page: params[:page]`